skyportal/tests/api/test_assignments.py

```diff
@@ -138,9 +138,5 @@ def test_group1_user_cannot_see_group2_assignment(
     assert data['status'] == 'success'
 
     status, data = api('GET', f'assignment/{id}', token=view_only_token)
-    assert status == 400
-    assert data['status'] == 'error'
-
-    status, data = api('GET', 'assignment/', token=view_only_token)
     assert status == 200
-    assert public_source_group2.id not in [a['id'] for a in data['data']]
+    assert data['status'] == 'success'
```

Comment:
I realize this is now what is reflected in the permissions defined in `models.py` but it does reflect a change from what the API used to do. Just want to confirm that we actually don't want to restrict the access by groups? It seem weird to me especially that update/delete on assignments is allowed for anyone with access to the object and observing run when say, the parent observing run has update/delete restricted to just the owner.

Reply:
Yes, this is correct. Observing runs are a bit peculiar as they are highly permissive. We need anyone to be able to update assignment state (e.g., observer A marked this object as observed, or marked a spectrum as taken). Observers can often come from all different places.